### PR TITLE
CSS: New Less Fathead

### PR DESCRIPTION
**Pull request by Lord-Lavios**
## Description of new Instant Answer, or changes
There are some small fixes left to do. IA : https://duck.co/ia/view/less

## Related Issues and Discussions
#764 

## People to notify
@pjhampton 

## Testing & Review
To be completed by Language Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [x] Instant Answer page is correctly filled out and contains:
    - One topic for the Search Space Language (Java, Python, Scala, Ruby, etc.)
    - One topic from: Reference, Help, Libraries, Tools
        - Documentation Fatheads are considered "Reference"
    - Description, Data source, and 2+ example queries
    - Perl Module (e.g. "DDG::Fathead::PerlDoc" -- we only need a name, not an actual file)
    - Source Name (for "More at <source_name>" link)
    - Source Domain (must contain http:// or https:// -- can be the same as Data Source)
    - Source Info (used as Subtitle for each Article -- usually matches the IA Name)
    - 'Skip Abstract' is checked off
    - Source ID (ping @moollaza to assign one, once Fathead is ready for Beta deploy)

**Code**
- [x] Uniformly indented, well commented
- [ ] Fetch.sh and Parse.sh run without errors
- [x] Output contains no blank lines, or multi-line entries
- [ ] Fathead Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/less

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/fathead/less/output.txt [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
...og__sub">Note</span><p> in versions before 2.2.0 the result is <code>base64</code> encoded .</p><span class="prog__sub">Generated background image has red color on the left, green at 30% of its width and ends with a blue color. Base64 encoded part contains following svg-gradient</span><p></p><pre><code class="lang-xml">&lt;?xml version=&quot;1.0&quot; ?&gt;\n&lt;svg xmlns=&quot;http://www.w3.org/2000/svg&quot; version=&quot;1.1&quot; width=&quot;100%&quot; height=&quot;100%&quot; viewBox=&quot;0 0 1 1&quot; preserveAspectRatio=&quot;none&quot;&gt;\n    &lt;linearGradient id=&quot;gradient&quot; gradientUnits=&quot;userSpaceOnUse&quot; x1=&quot;0%&quot; y1=&quot;0%&quot; x2=&quot;100%&quot; y2=&quot;0%&quot;&gt;\n        &lt;stop offset=&quot;0%&quot; stop-color=&quot;#ff0000&quot;/&gt;\n        &lt;stop offset=&quot;30%&quot; stop-color=&quot;#008000&quot;/&gt;\n        &lt;stop offset=&quot;100%&quot; stop-color=&quot;#0000ff&quot;/&gt;\n    &lt;/linearGradient&gt;\n    &lt;rect x=&quot;0&quot; y=&quot;0&quot; width=&quot;1&quot; height=&quot;1&quot; fill=&quot;url(#gradient)&quot; /&gt;\n&lt;/svg&gt;</code></pre></section>	http://lesscss.org/functions/#misc-functions-svg-gradient
```
Comment (moollaza) [inline lib/fathead/less/output.txt]: Whoops looks like the SVG is wrapped in a `<pre><code>` block -- I don't think we want that

File: lib/fathead/less/output.txt [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
...i></ul><span class="prog__sub">Parameters</span><p></p><ul class="prog__ul"><li><code>string</code>: format string with placeholders,</li><li><code>anything</code>* : values to replace placeholders.</li></ul><span class="prog__sub">Returns</span><p> formatted <code>string</code>.</p><span class="prog__sub">Example</span><p></p><pre><code class="lang-less">format-a-d: %(&quot;repetitions: %a file: %d&quot;, 1 + 2, &quot;directory/file.less&quot;);\nformat-a-d-upper: %(&apos;repetitions: %A file: %D&apos;, 1 + 2, &quot;directory/file.less&quot;);\nformat-s: %(&quot;repetitions: %s file: %s&quot;, 1 + 2, &quot;directory/file.less&quot;);\nformat-s-upper: %(&apos;repetitions: %S file: %S&apos;, 1 + 2, &quot;directory/file.less&quot;);</code></pre><span class="prog__sub">Output</span><p></p><pre><code class="lang-css">format-a-d: &quot;repetitions: 3 file: &quot;directory/file.less&quot;&quot;;\nformat-a-d-upper: &quot;repetitions: 3 file: %22directory%2Ffile.less%22&quot;;\nformat-s: &quot;repetitions: 3 file: directory/file.less&quot;;\nformat-s-upper: &quot;repetitions: 3 file: directory%2Ffile.less&quot;;</code></pre></section>	http://lesscss.org/functions/#string-functions-% format
```
Comment (moollaza) [inline lib/fathead/less/output.txt]: `string functions` is a little too generic. The categories have no context so we need more specific names. Let's use `LESS String Functions` (Title Casing would be nice too)

**Comment by moollaza** [comment]
@Lord-Lavios Thanks! This LGTM :+1:

Please add a "trigger_words.txt" file with `less`, `lesscss` and `less css` -- I think we may need to require those trigger words to be in the query as most of these article titles are very generic (e.g. "green") and we should only show those when one of the trigger words is in the query

File: lib/fathead/less/output.txt (hunk 1 above)
Comment (Lord-Lavios) [inline lib/fathead/less/output.txt]: @moollaza I'm not sure what to do with it. should I delete it?

**Pull request merged**